ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-lonely-if */
```

Comment:
I believe you might be able to correct the if logic without disabling eslint (on the first view I think the ifs after line 365 are causing the check to fail). I would avoid at all costs to disable eslint rules.